client/rpc/src/eth.rs

```diff
@@ -54,6 +68,7 @@ pub struct EthApi<B: BlockT, C, P, CT, BE, H: ExHashT> {
 	network: Arc<NetworkService<B, H>>,
 	is_authority: bool,
 	signers: Vec<Box<dyn EthSigner>>,
+	overrides: BTreeMap<EthereumStorageSchema, Box<dyn StorageOverride<B> + Send + Sync>>,
```

Comment:
Consider changing this to `BTreeMap<Option<EthereumStorageSchema>, Box<dyn StorageOverride<B> + Send + Sync>>` -- that is, with an `Option` of the key. Then, use `None` as the fallback. That will simplify the code below.

The above would make it invalid if the struct is initialized without `None`, so we need to check that.

Another option is to define an explicit `fallback` field and make it of type `Box<dyn StorageOverride<B> + Send + Sync>`. Both of the options seem good to me.

Reply:
I had included a separate `fallback` field originally. I decided to remove it because doing so would save the overhead of a dynamic dispatch in the case that we are falling back. Maybe that tiny optimization is not worth the added code in all the methods.

Reply:
I agree. I don't think that tiny optimization is worth it. Plus, once we switched to use schema, the fallback should be rarely activated.

Reply:
done